src/styles/custom/_forms.scss

```diff
@@ -64,6 +64,11 @@ span.file-title {
 }
 // end of input file type styling
 
+// General FORM styling needs
+form {
```

Comment:
why is this needed? If it is a styling update we should have a separate ticket for it.

Reply:
agreed, not necessary at this point, will remove this from pr.

Reply:
ok, was having issue removing this from this pr's files directly on github, so instead updated locally and pushed.